sidecar/init.go

```diff
@@ -44,6 +50,61 @@ func NewInitCommand(cfg *Config) *cobra.Command {
 	return cmd
 }
 
+// checke leader or service
+func CheckServiceExist(cfg *Config, service string) bool {
+	serviceURL := fmt.Sprintf("http://%s-%s:%v%s", cfg.ClusterName, service, utils.XBackupPort, "/health")
```

Comment:
We can package this to `internal` package like xenon executor.

Reply:
internal do not have  `timeout `